distribution/ddf-common/src/main/resources/etc/artemis.xml

```diff
@@ -54,7 +57,7 @@
         <network-check-URL-list>http://www.apache.org</network-check-URL-list> -->
 
         <network-check-period>10000</network-check-period>
-        <network-check-timeout>255</network-check-timeout>
+        <network-check-timeout>1000</network-check-timeout>
```

Comment:
revert this back to 255 it is the max that linux supports add a comment to that effect here also for me.

Reply:
fixed